### PR TITLE
moar submission + user docs

### PR DESCRIPTION
**Pull request by rosibaj**
- added basic introduction and submission steps 
- updated user management and roles with draft 
- removed docs not needed from aleks 
-

### CODE REVIEW

File: docs/registering-samples.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The basic set of data that must be registered for each sample consists of:
 * Submitter Sample ID
 * Sample Type
 
-During sample registration, **ARGO IDs** will be assigned to your program data. Any attempts to submit data  which does not refer to a previously registered samples will result in an error, i.e you will be prompted to complete sample registration before any clinical or molecular data is submitted.
+During sample registration, **ARGO IDs** will be assigned to your program data. Any attempts to submit data that does not refer to a previously registered samples will result in an error. You will be prompted to complete sample registration before any clinical or molecular data is submitted.
```
Comment (joneubank) [inline docs/registering-samples.md]: "a previously registered samples" - singular or plural?

File: docs/submission-getting-started.md [hunk 2]
```diff
@@ -3,4 +3,15 @@ id: submission-getting-started
 title: Getting Started
 ---
 
-Placeholder
+
+After receiving your invitation email to the ARGO Platform from the Data Coordination Center (DCC), you are ready to get started submitting data to the ARGO Platform! Submission of data can start as soon as you complete your registration.
+
+## Submitting Data to the ARGO Platform
+1. Begin by [granting access](managing-program-Access) to your submission area to your colleagues. Depending on the role you wish others to take in submission, you will be asked to select permission based roles for any users added to your program.
```
Comment (joneubank) [inline docs/submission-getting-started.md]: not sure if `managing-program-Access` will have link issues due to capitalization... probably not an issue but either go lowercase or double check that it works.

**Comment by rosibaj** [comment]
@joneubank Thanks  - just needed the functional issues.  I'll re-edit once i can get screenshots in as well!

File: docs/submission-getting-started.md [hunk 3]
```diff
@@ -3,4 +3,15 @@ id: submission-getting-started
...
+1. Data submissions begin with [Registering Samples](registering-samples).  This step is mandatory before any clinical or molecular submissions. Once sample have been registered, the following steps can be completed in any order.
```
[the remark this answers is not in the document]
Comment (rosibaj) [reply docs/submission-getting-started.md]: done

File: docs/submission-getting-started.md (hunk 2 above)
Comment (rosibaj) [reply docs/submission-getting-started.md]: done

File: docs/submission-getting-started.md [hunk 4]
```diff
@@ -3,4 +3,15 @@ id: submission-getting-started
...
+1. Use the workspace to [Submit Clinical Data](submitting-clinical-data) related to donors.  
+1. Use your user token and data management tools Song and Score to [Submit Molecular Data](submitting-molecular-data) to your assigned processing center.  Processing of genomic data will being once you signal a batch is ready to begin.
+
+Once clinical and molecular data has been submitted
```
[the remark this answers is not in the document]
Comment (rosibaj) [reply docs/submission-getting-started.md]: will fix later once we have the genomic docs - leave for  now

File: docs/registering-samples.md (hunk 1 above)
Comment (rosibaj) [reply docs/registering-samples.md]: fixed

**Pull request merged**